flask_meld/component.py

```diff
@@ -196,6 +196,9 @@ def updated(self, name):
     def render(self, component_name):
         return self._view(component_name)
 
+    def _render_template(self, template_name: str, context_variables: dict):
+        return render_template(template_name, **context_variables)
+
```

Comment:
extracted the template rendering, so a Custom Component can overwrite the template rendering. Makes tests way easier.

Saving the component name in the component itself could make this code even better (regarding OOP)